File: java/PrimeNumber.java

import java.util.Scanner;

public class PrimeNumber{

    public static  boolean isPrime(int num){
        boolean flag= true;
         for(int i=2;i<=num/2;i++){
            if(num%i==0){
                flag=false;
            }
         }
         return flag;
    }
   public static void main(String[] args) {
       Scanner sc=new Scanner(System.in);
       System.out.println("Enter the number");
       int num=sc.nextInt();
       if(isPrime(num)){
        System.out.println(num+ " is a prime number");
       }
       else{
        System.out.println(num+ " is not a prime number");
       }
   }

}

File: python/Prime Number.py
n = 11
if n <= 1:
    print(False)
else:
    is_prime = True  # Flag variable
    for i in range(2, int(n**0.5) + 1):
        if n % i == 0:
            is_prime = False
            break
    print(is_prime)